client/src/components/application/Examine/CompName.vue

```diff
@@ -90,7 +92,10 @@
                 <span class="name-state-icon" v-html="setIcon(compName3.state)"></span>
                 <button class="btn btn-undo" v-if="is_undoable_3"
                         v-on:click="undoDecision(3)">Undo Decision</button>
-                <span class="decision-text">{{ compName3.decision_text }}</span>
+                <span class="decision-text"
+                      v-bind:class="{completed: is_complete}">{{ compName3.decision_text }}</span>
+              </td>
+                <span v-bind:class="{completed: is_complete, 'decision-text': !is_complete }">{{ compName3.decision_text }}</span>
```

Comment:
This second guy snuck in - compName3.decision_text - it's correctly on line 96, incorrectly on line 98.

Reply:
Oooh with an extra /td as well

Reply:
ooops i fixed that forgot to push